.gitignore

```diff
@@ -152,6 +157,7 @@ node_modules
 npm-debug.log*
 derby.log
 metastore_db
+yarn.lock
```

Comment:
We do store yarn.lock file: https://github.com/apache/airflow/blob/master/airflow/www/yarn.lock

Reply:
Yes I notice this one.
What I'd like to ignore is one in the root of the source tree.
But I noticed, the yarn.lock in the root is mistakenly generated ( Maybe, I have run `yarn` command at the root...),
Anyway, I revert this piece of change and modify the description.
Thanks.